.eslint-resolver.js

```diff
@@ -0,0 +1,22 @@
+const path = require('path');
+const resolve = require('resolve')
+
```

Comment:
nit: could use a link to the documentation for the interface this file implements
```suggestion

// Implements the following resolver spec:
// https://github.com/benmosher/eslint-plugin-import/blob/master/resolvers/README.md
```